src/BeanstalkClient.php

```diff
@@ -99,7 +98,8 @@ public function pause(string $tube, int $delay): Promise {
         });
     }
 
-    public function put(string $payload, int $timeout = 60, int $delay = 0, $priority = 0): Promise {
+    public function put(string $payload, int $timeout = 60, int $delay = 0, $priority = 0)
```

Comment:
We should add return types to all of these.

Reply:
I don't know what the return types are. In most cases it seems they would just be `mixed` anyway.